zerver/models.py

```diff
@@ -723,6 +739,21 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
     # they will not be available regardless of users' personal settings.
     enable_read_receipts = models.BooleanField(default=False)
 
+    realm_permission_group_settings: Dict[str, GroupPermissionSetting] = {
+        "direct_message_initiator_group": GroupPermissionSetting(
+            require_system_group=True,
+            allow_internet_group=False,
+            allow_owners_group=True,
+            allow_nobody_group=True,
+        ),
+        "direct_message_permission_group": GroupPermissionSetting(
+            require_system_group=True,
+            allow_internet_group=False,
+            allow_owners_group=True,
+            allow_nobody_group=True,
+        ),
```

Comment:
Given how much "remove _id" parsing you have elsewhere in this PR, I wonder if we should just provide two objects; `realm_permission_group_setting_fields` and `realm_permission_group_setting_id_fields`, with one constructed from the other by appending `_id` to the end of all the keys, so that code can access whichever is most convenient for it?